pkg/server/container_create.go

```diff
@@ -225,6 +226,18 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	if username := securityContext.GetRunAsUsername(); username != "" {
 		specOpts = append(specOpts, oci.WithUsername(username))
 	}
+	if gid := securityContext.GetRunAsGroup(); gid != nil {
+		if securityContext.GetRunAsUser() == nil || securityContext.GetRunAsUsername() == "" {
+			return nil, errors.New("user group is specified without user")
```

Comment:
maybe output uid and username here so we can see if one or both are nil...

Reply:
Will do.